### PR TITLE
[UT] Add unstructed helper uinttest

### PR DESCRIPTION
**Pull request by qingwave**
Signed-off-by: qingwave <isguory@gmail.com>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Add uinttest for util/helper/unstructed.go

**Which issue(s) this PR fixes**:
Subtask of #2579

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

### CODE REVIEW

File: pkg/util/helper/unstructured_test.go [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+					"metadata": map[string]interface{}{
+						"name": "some-pod",
+					},
+				},
+			},
+			out:         &corev1.Pod{},
+			expectedErr: false,
+		},
+		{
+			name: "convert unstructured object success",
```
Comment (jwcesign) [inline pkg/util/helper/unstructured_test.go:47]: looks like it's duplicated with the previous one.

Comment (qingwave) [reply pkg/util/helper/unstructured_test.go]: Thanks for your review, remove the duplicated case

**Review by RainbowMango — APPROVED** [review]
/approve

**Pull request merged**